sdk/storage/azblob/service/client.go

```diff
@@ -82,6 +83,23 @@ func NewClientFromConnectionString(connectionString string, options *ClientOptio
 	return NewClientWithNoCredential(parsed.ServiceURL, options)
 }
 
+// GetUserDelegationCredential obtains a UserDelegationKey object using the base ServiceURL object.
+// OAuth is required for this call, as well as any role that can delegate access to the storage account.
+func (s *Client) GetUserDelegationCredential(ctx context.Context, info generated.KeyInfo, o *GetUserDelegationKeyOptions) (*UserDelegationCredential, error) {
```

Comment:
`generated.KeyInfo` isn't a publicly available type.  This will need a type alias.

Reply:
Looks like the type alias already exists, just need to change the param type here.